app/src/main/kotlin/com/wabadaba/dziennik/di/ApplicationModule.kt

```diff
@@ -67,4 +70,10 @@ class ApplicationModule(private val mainApplication: Application) {
     @Provides
     @Singleton
     fun provideGPServicesChecker(): GPServicesChecker = GPServicesChecker()
+
+    @Provides
+    fun provideNotificationManager(context: Context) = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
```

Comment:
I see no point in having NotificationManager provided by Dagger. Unless it helps with testing, please put this inside LibrusGcmListenerService